From e7d2878fa7866e4e44b408b1b23c6b02966a0fc9 Mon Sep 17 00:00:00 2001
From: Satya Durga Srinivasu Prabhala <satyap@codeaurora.org>
Date: Tue, 24 Nov 2020 01:09:06 -0800
Subject: [PATCH 178/830] ANDROID: sched: add missing data type for cpu
 variable
X-NVConfidentiality: public

While bringing in a change from older Kernel, commit 3adfd8e344ac328
("ANDROID: sched: avoid placing RT threads on cores handling softirqs")
missed to add data type for cpu variable. Fix it by adding data type.

Bug: 168521633
Change-Id: I4cd3d0b68b5962004f295ce8d07546b2067bc728
Signed-off-by: Satya Durga Srinivasu Prabhala <satyap@codeaurora.org>
---
 kernel/sched/cpupri.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/sched/cpupri.c b/kernel/sched/cpupri.c
index 051df5be7d44..69bc0de4d2eb 100644
--- a/kernel/sched/cpupri.c
+++ b/kernel/sched/cpupri.c
@@ -49,7 +49,7 @@ static int convert_prio(int prio)
 static void
 drop_nopreempt_cpus(struct cpumask *lowest_mask)
 {
-	unsigned cpu = cpumask_first(lowest_mask);
+	unsigned int cpu = cpumask_first(lowest_mask);
 	while (cpu < nr_cpu_ids) {
 		/* unlocked access */
 		struct task_struct *task = READ_ONCE(cpu_rq(cpu)->curr);
-- 
2.17.1

